CHANGELOG.md

```diff
@@ -52,15 +110,15 @@ _Changes since release/1.7.0-beta01_
   - [Material 1.7.0-rc01](https://developer.android.com/jetpack/androidx/releases/compose-material#1.7.0-rc01)
   - [Material3 1.3.0-rc01](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.3.0-rc01)
 
-- Lifecycle libraries `org.jetbrains.androidx.lifecycle:lifecycle-*:2.8.2`. Based on [Jetpack Lifecycle 2.8.4](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.4)
+- Lifecycle libraries `org.jetbrains.androidx.lifecycle:lifecycle-*:2.8.3-rc01`. Based on [Jetpack Lifecycle 2.8.5](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.5)
```

Comment:
I guess we should keep previous releases as-is

Reply:
We always mentioned alpha/beta/rc here. No difference with hotfix-rc